### PR TITLE
HRP-14 [FrontEnd] My Profile Page

### PR DESCRIPTION
**Pull request by Funkoala14**
view my profile, edit profile info ( except visa, docs), no input validation yet

### CODE REVIEW

File: client/vite.config.js [hunk 1]
```diff
@@ -19,7 +24,7 @@ export default defineConfig({
     server: {
         port: 3000,
         proxy: {
-            '/api': 'http://localhost:5000', // backend
+            '/vi/api': 'http://localhost:5000', // backend
```
Comment (tuho9428) [inline client/vite.config.js]: Should it be "v1" instead of "vi"?

**Review by Funkoala14 — COMMENTED** [review]
reviewed

File: client/vite.config.js (hunk 1 above)
Comment (Funkoala14) [reply client/vite.config.js]: you are right, I fixed it

Comment (tuho9428) [reply client/vite.config.js]: Did you try to use `git rebase dev`?

**Pull request closed without merging**